CMakeLists.txt

```
@@ -111,6 +108,14 @@ if(BUILD_PYTHON_INTERFACE)
   find_package(Boost REQUIRED COMPONENTS system)
 endif(BUILD_PYTHON_INTERFACE)
 
+if(Boost_VERSION_STRING VERSION_LESS 1.82)
```

```suggestion
if(Boost_VERSION_STRING VERSION_LESS 1.81)
```

This would allow us to test it with Boost 1.81 and get rid of the warnings